src/Tribe/Customizer/Global_Elements.php

```diff
@@ -476,49 +478,68 @@ public function setup() {
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
 
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
+			$manager,
+			$customizer->get_setting_name( 'accent_color_heading', $section ),
+			[
+				'label'   => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
+
 		$manager->add_setting(
 			$customizer->get_setting_name( 'accent_color', $section ),
-			array(
-				'default'              => '#334AFF',
-				'type'                 => 'option',
-
+			[
+				'default' => '#334AFF',
+				'type'    => 'option',
 				'sanitize_callback'    => 'sanitize_hex_color',
 				'sanitize_js_callback' => 'maybe_hash_hex_color',
-			)
+			]
 		);
 
 		$manager->add_control(
 			new WP_Customize_Color_Control(
 				$manager,
 				$customizer->get_setting_name( 'accent_color', $section ),
-				array(
-					'label'   => esc_html__( 'Accent Color', 'the-events-calendar' ),
-					'section' => $section->id,
-				)
+				[
+					'label'       => esc_html__( 'Accent Color', 'the-events-calendar' ),
+					'section'     => $section->id,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'section'" and double arrow, but found 5.

<sub>hash: 045e4e33a1138c5535bd4ce9519652b1</sub>